client/src/components/categorieslist/CategoriesList.jsx

```diff
@@ -10,7 +11,7 @@ import "swiper/css";
 import "swiper/css/navigation";
 import "./CategoriesList.css";
 
-export default function CategoriesList({ category }) {
+export default function CategoriesList({ category, reload = false }) {
```

Comment:
question: why do we need a reload here?

Reply:
# What ?

we're passing this boolean from `VideoPage` down to the `VideoCard` component to set the `NavLink`'s `reloadDocument` prop to `true`

## Why ?

simply navigating to another video from the "same category" links (on the bottom of `VideoPage`) causes some elements to not reload. While we could track down and try to figure out what to manually refresh with states or otherwise, the simplest solution is simply to transform the `NavLink` into a regular link (`a`) so that it reloads the page when switching videos from `VideoPage` using the [reloadDocument](https://reactrouter.com/en/main/components/nav-link#reloaddocument) props of `NavLink`.